pkg/virt-handler/vm.go

```diff
@@ -16,10 +17,21 @@ func NewVMController(listWatcher cache.ListerWatcher, domainManager libvirt.Doma
 			fmt.Printf("VM ADD\n")
 			vm := obj.(*v1.VM)
 			err := domainManager.SyncVM(vm)
+
 			if err != nil {
 				fmt.Println(err)
 				recorder.Event(vm, kubev1.EventTypeWarning, v1.SyncFailed.String(), err.Error())
 				return cache.ErrRequeue{Err: err}
+			} else {
+				restClient, err := kubecli.GetRESTClient()
+				if err != nil {
+					return cache.ErrRequeue{Err: err}
+				}
+				vm.Status.Phase = v1.Succeeded
+				err = restClient.Put().Resource("vms").Body(vm).Name(vm.ObjectMeta.Name).Namespace(kubeapi.NamespaceDefault).Do().Error()
```

Comment:
@rmohr do you actually know if the client will put the complete serialized entity again?

/me wonders if there is something like a partial update for just a single field.